Test/git-issues/git-issue-1963a.dfy

```diff
@@ -0,0 +1,17 @@
+// RUN: %dafny /compile:0 "%s" > "%t"
+// RUN: %diff "%s.expect" "%t"
+
+/// This file checks that refinement is forbidden for newtypes
+
+module A {
+  newtype NT = x: int | x >= 0 witness 0
+  lemma P(x: NT) ensures x >= 0 {}
+}
+
+module B refines A {
+  newtype NT = ... x: int | x < 0 witness -1
```

Comment:
I suggest you add a comment on this line, saying that we expect an error. This makes it easier to read the test files (without looking up every error in the corresponding .expect file).

Reply:
```suggestion
  newtype NT = ... x: int | x < 0 witness -1 // Error: Change (or repeat) a newtype body
```